plugins/woocommerce-blocks/assets/js/blocks/add-to-cart-with-options/edit.tsx

```diff
@@ -0,0 +1,92 @@
+/**
+ * External dependencies
+ */
+import { useEffect } from '@wordpress/element';
+import { InnerBlocks, useBlockProps } from '@wordpress/block-editor';
+import { BlockEditProps } from '@wordpress/blocks';
+import { useSelect } from '@wordpress/data';
+import { isSiteEditorPage } from '@woocommerce/utils';
+import { getSettingWithCoercion } from '@woocommerce/settings';
+import { isBoolean } from '@woocommerce/types';
+
+/**
+ * Internal dependencies
+ */
+import { useIsDescendentOfSingleProductBlock } from '../../atomic/blocks/product-elements/shared/use-is-descendent-of-single-product-block';
+import { QuantitySelectorStyle, AddToCartFormSettings } from './settings';
+import { INNER_BLOCKS_TEMPLATE } from './constants';
+export interface Attributes {
+	className?: string;
+	isDescendentOfSingleProductBlock: boolean;
+	quantitySelectorStyle: QuantitySelectorStyle;
+}
+
+export type FeaturesKeys =
+	| 'isStepperLayoutFeatureEnabled'
+	| 'isBlockifiedAddToCart';
+
+export type FeaturesProps = {
+	[ key in FeaturesKeys ]?: boolean;
+};
+
+export type UpdateFeaturesType = ( key: FeaturesKeys, value: boolean ) => void;
+
+// Pick the value of the "blockify add to cart flag"
+const isBlockifiedAddToCart = getSettingWithCoercion(
+	'isBlockifiedAddToCart',
+	false,
+	isBoolean
+);
+
+const AddToCartFormEdit = ( props: BlockEditProps< Attributes > ) => {
+	const { setAttributes } = props;
+
+	const isStepperLayoutFeatureEnabled = getSettingWithCoercion(
```

Comment:
The stepper feature doesn't belong to this block. It belongs to the quantity selector block [#53009](https://github.com/woocommerce/woocommerce/issues/53009)